ophyd/userapi/scan_api.py

```
@@ -491,16 +386,21 @@ def _make_log_entry(self):
 
         msg.append('{0:=^80}'.format(''))
 
-        for p in self.positioners + self.triggers + self.detectors:
+        for p in self.positioners:
             try:
                 msg.append('PV:{}'.format(p.report['pv']))
-            except KeyError:
+            except:
+                pass
+
+        for p in self.detectors:
+            try:
+                [msg.append(val) for key, val in p.source().iteritems()]
+            except:
```

And another catch-all exception here.